src/screens/map/MapScreen.styles.ts

```diff
@@ -1,7 +1,30 @@
+import { COLORS } from '@nightlight/src/global.styles';
 import { StyleSheet } from 'react-native';
 
 export default StyleSheet.create({
   container: {
     height: '100%',
   },
+  mapControlButton: {
+    backgroundColor: COLORS.NIGHTLIGHT_GRAY,
+    borderColor: COLORS.NIGHTLIGHT_BLACK,
+    position: 'absolute',
+    borderStyle: 'solid',
+    borderRadius: 10,
+    borderWidth: 2.5,
+    width: 50,
+    height: 50,
+  },
+  currentLocationButton: {
+    bottom: 140,
+    right: 10,
```

Comment:
use margins instead

Reply:
margins won't work because we're using position absolute. sticking with bottom/right